docs/admin/federation-apiserver.md

```diff
@@ -1,8 +1,3 @@
 This file is autogenerated, but we've stopped checking such files into the
 repository to reduce the need for rebases. Please run hack/generate-docs.sh to
 populate this file.
-
```

Comment:
I don't know why these are removed from documentation. It is hard to tell because the commit has both manual and automated changes. Did you change anything other than adding a comment line? which file?